charts/policy-reporter/templates/service.yaml

```diff
@@ -1,21 +1,32 @@
+{{- $apiEnabled := .Values.api.enabled -}}
+{{- $uiEnabled := .Values.ui.enabled -}}
+{{- if .Values.service.enabled -}}
 apiVersion: v1
 kind: Service
 metadata:
   name: {{ include "policyreporter.fullname" . }}
   labels:
     {{- include "policyreporter.labels" . | nindent 4 }}
+{{- range $key, $value := $.Values.service.labels }}
+    {{ $key }}: {{ $value | quote }}
+{{- end }}
+  annotations:
+{{- range $key, $value := $.Values.service.annotations }}
+    {{ $key }}: {{ $value | quote }}
+{{- end }}
 spec:
-  type: ClusterIP
+  type: {{ .Values.service.type }}
   ports:
-    - port: 2112
+    - port: {{ .Values.service.port }}
       targetPort: http
       protocol: TCP
       name: http
-    {{- if or .Values.api.enabled .Values.ui.enabled }}
-    - port: 8080
```

Comment:
this part is a bit problemetic because this port is currently hard coded in the UI as backend port because I can't access a variable from the parent chart in the sub chart. If you change the default you have to manually set the backend URL in the UI

Reply:
https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#global-chart-values
replace to `global`
Only one problem, if helm ReleaseName is different then backend name should be replaced as well
I can't replace that URL automatically

Reply:
fine for now. I will improve the documentation this week and add this behavior.